src/cascadia/TerminalControl/InteractivityAutomationPeer.cpp

```diff
@@ -35,14 +35,19 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         THROW_IF_FAILED(::Microsoft::WRL::MakeAndInitialize<::Microsoft::Terminal::TermControlUiaProvider>(&_uiaProvider, _interactivity->GetRenderData(), this));
     };
 
+    // Bounds is expected to be in pixels.
     void InteractivityAutomationPeer::SetControlBounds(const Windows::Foundation::Rect bounds)
     {
-        _controlBounds = til::rect{ til::math::rounding, bounds };
+        _controlBounds = { til::math::rounding, bounds };

```

Comment:
should we rename these to be like `_controlBoundsPx` and `_controlPaddingDip`?

Reply:
see even I got it backwards - it looks like we actually store control padding as PX despite the comment below saying "in dips" (i understand now that it means "the parameter is in dip")